system-addon/lib/TelemetryFeed.jsm

```diff
@@ -274,6 +278,32 @@ this.TelemetryFeed = class TelemetryFeed {
   }
 
   /**
+   * handlePagePrerendered - Set the session as prerendered
+   *
+   * @param  {string} portID the portID of the target session
+   */
+  handlePagePrerendered(portID) {
+    const session = this.sessions.get(portID);
+
+    if (!session) {
+      // It's possible the tab was never visible – in which case, there was no user session.
+      return;
+    }
+
+    session.perf.is_prerendered = true;
+  }
+
+  /**
+   * handleNewTabInit - Handle NEW_TAB_INIT
+   *
+   * @param  {obj} action the Action object
+   */
+  handleNewTabInit(action) {
+    const session = this.addSession(au.getPortIdOfSender(action), action.data.url);
+    session.perf.is_preloaded = action.data.browser === action.data.browser.ownerGlobal.gBrowser._preloadedBrowser;
```

Comment:
Oh, this is probably an easier check ;)
https://searchfox.org/mozilla-central/search?q=%22isPreloadBrowser%22&path=browser

`browser.getAttribute("isPreloadBrowser") == "true"`